deploy_polygon/02_asset/02_deploy_asset_signed_auction_auth.ts

```diff
@@ -0,0 +1,35 @@
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
+import {DeployFunction} from 'hardhat-deploy/types';
+// import {skipUnlessTest} from '../../utils/network';
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const {deployments, getNamedAccounts, getUnnamedAccounts} = hre;
+  const {deploy} = deployments;
+
+  const {
+    deployer,
+    assetAdmin,
+    assetAuctionFeeCollector,
+  } = await getNamedAccounts();
+  const others = await getUnnamedAccounts();
+
+  const assetContract = await deployments.get('PolygonAsset');
+
+  await deploy('AssetSignedAuctionAuth', {
+    from: deployer,
+    contract: 'AssetSignedAuctionAuth',
+    args: [
+      assetContract.address,
+      assetAdmin,
+      others[0],
+      assetAuctionFeeCollector,
+      200,
+    ],
+    log: true,
+    skipIfAlreadyDeployed: true,
+  });
```

Comment:
In the L1 deployment, a lot of stuffs is done after the `deploy` regarding the `feeEvents`

Does we need that here too ?